.github/workflows/test.yml

```
@@ -18,7 +18,18 @@ jobs:
         fetch-depth: 0
         # fetch PR commit, not predicted merge commit
         ref: ${{ github.event.pull_request.head.sha }}
-    - uses: cachix/install-nix-action@v31
+
+    - if: matrix.os == 'ubuntu-latest' || matrix.os == 'ubuntu-24.04-arm'
```

```suggestion
    - if: ${startsWith(matrix.os, 'ubuntu-')}
```
Similarly in other places too!